llvm/lib/Transforms/Vectorize/VPlanUnroll.cpp

```diff
@@ -264,6 +264,13 @@ void UnrollState::unrollRecipeByUF(VPRecipeBase &R) {
     return;
 
   if (auto *VPI = dyn_cast<VPInstruction>(&R)) {
+    VPValue *Op0, *Op1;
+    if (match(VPI, m_VPInstruction<VPInstruction::ResumePhi>(m_VPValue(Op0),
```

Comment:
How/Is this change related to this patch, ResumePhi's for FORs are used by other than part zero?

Reply:
This is related to removing VPLiveOut, VPIRInstructions aren't marked as using the first lane part/lane (technically they would use the last part/lane?)

Reply:
VPInstruction::ResumePhi `isSingleScalar()`, so should probably qualify as only first part used below?